net/node/infoUpdate.go

```diff
@@ -104,10 +104,30 @@ func (node *node) ReqNeighborList() {
 }
 
 func (node *node) ConnectSeeds() {
-	if node.nbrNodes.GetConnectionCnt() == 0 {
+	if node.nbrNodes.GetConnectionCnt() < MINCONNCNT {
 		seedNodes := config.Parameters.SeedList
 		for _, nodeAddr := range seedNodes {
-			go node.Connect(nodeAddr)
+			established := false
+			var n Noder
+			var ip net.IP
+			node.nbrNodes.Lock()
+			for _, tn := range node.nbrNodes.List {
+				addr := getNodeAddr(tn)
+				ip = addr.IpAddr[:]
+				addrstring := ip.To16().String() + ":" + strconv.Itoa(int(addr.Port))
+				if tn.GetState() == ESTABLISH && nodeAddr == addrstring {
```

Comment:
1.  I mean if a node have the state of HANDSHAKE, HAND, INIT ,  it means that it is connecting now, so calling node.Connect(nodeAddr) is not necessary.
2.  I mean you can call ReqNeighborList on node2， node3， not just restricted to the seednode.